tests/models/test_pytorch_models.py

```diff
@@ -101,7 +101,12 @@ class TestPyTorchModel:
             if inspect.isclass(obj) and name != "Optimizer"
         ],
     )
-    def test_save(self, tmpdir, net, optimizer):
+    def test_save(self, mocker, tmpdir, net, optimizer):
+        mocker.patch(
+            "tiledb.ml.models.pytorch.PyTorchTileDBModel._use_legacy_schema",
+            return_value=False,
+        )
```

Comment:
You may try using a [pytest session fixture](https://stackoverflow.com/a/65928873) to have this patch being applied automatically to every test.